grammar.hgr

```diff
@@ -3,7 +3,7 @@ grammar {
     partials {
       r'[a-zA-Z]' -> _identifier_start
       r'[a-zA-Z0-9_]' -> _identifier_follow
-      r'(Array|Map|Object|Pair|Boolean|Int|Float|Uri|File|String)(?!{%_identifier_follow%})' -> _type
```

Comment:
I believe edits on the grammar.hgr were unintentional (as a result of my misuse of Github). I will attempt to fix it.